src/projects/markdown-previewer/index.jsx

```diff
@@ -14,11 +14,15 @@ marked.setOptions({
   breaks: true
 });
 
-// INSERTS target="_blank" INTO HREF TAGS (required for codepen links)
+// INSERTS target="_blank" INTO HREF TAGS (required for Codepen links)
 const renderer = new marked.Renderer();
-renderer.link = function(href, title, text) {
+renderer.link = function (href, title, text) {
   return `<a target="_blank" href="${href}">${text}` + '</a>';
 };
+// Use Prism for code highlighting
+renderer.code = function (code) {
+  return `<pre><code class="language-javascript">${code}` + `</code></pre>\n`;
```

Comment:
Very minor nitpick, but this could probably be simplified to:

```suggestion
  return `<pre><code class="language-javascript">${code}</code></pre>\n`;
```